### ISSUE TITLE
how to get label of segmentations in SynthText

### ISSUE THREAD

**Issue by TriplePool** [MhLiao/MaskTextSpotter#25]
Thank you for your work.
Could you please release the code of pre-processing of SynthText.

**Comment by MhLiao** [comment]
@TriplePool I will release the code of pre-processing of SynthText later. I hope the converted [annotations](https://1drv.ms/u/s!ArsnjfK83FbXgb5vgOOVPYywgCWuQw?e=UPuNTa) are helpful to you currently.

**Comment by TriplePool** [comment]
@MhLiao Thank you very much.

**Comment by TriplePool** [comment]
@MhLiao Hi, I have downloaded the converted annotations, and the label of segmentations is a quadrangle that from "wordBB" in SynthText instead of polygon like TotalText.
Is it necessary that pre-train the model with SynthText polygons label of word region, if I want to handle with curved  text?

**Comment by MhLiao** [comment]
@TriplePool In my opinion, more accurate annotations would be better. But I did not try that. If you do care about the performance of curved text, you can try it.

**Comment by ChChwang** [comment]
@MhLiao the char annotation of icdar2013 was used in training?

**Comment by TriplePool** [comment]
@MhLiao Got it, thank you.

**Comment by TriplePool** [comment]
> @TriplePool In my opinion, more accurate annotations would be better. But I did not try that. If you do care about the performance of curved text, you can try it.

Got it, thank you

**Comment by TriplePool** [comment]
> @MhLiao the char annotation of icdar2013 was used in training?

As far as I know, there is no char annotation in icdar2013 dataset, is there?

**Comment by MhLiao** [comment]
@TriplePool @ChChwang The char annotation of icdar2013 was used in training.

**Comment by ChChwang** [comment]
@MhLiao the char annotation was None in the training.  Is there error in the training code?

**Comment by ChChwang** [comment]
@MhLiao there are 858750  SynthText polygons labels ? but the num of imgs is 1717500.

**Comment by ChChwang** [comment]
@MhLiao c_class = self.char2num(strs[1:]) in the load_gt_from_txt, why the strs begin from 1?

**Comment by ChChwang** [comment]
@MhLiao there is error in maskrcnn_benchmark/data/datasets/synthtext.py, what if "," in labels?
see the label of city_112_47.jpg.

**Comment by ustczhouyu** [comment]
Please help me. After I use other methods to predict the results, I get .txt such as res_img_1.txt, and each line is x1, y1, x2, y2, x3, y3, x4, y4, word. I don't know how to generate S, W, and G results. How to evaluate the end-to-end result?  Please help me.
